cases/aws_instance/README.rst

```diff
@@ -33,6 +33,24 @@ Unsupported attributes
 * ``credit_specification``
 * ``private_ip``
 
+Special notes
+~~~~~~~~~~~~~
+This resource supports ``tags`` attribute
+
+Example tag
+
+``resource "aws_instance" "test" {``
```

Comment:
please use ```code block``` directive for highlighting code samples:
https://documentation.help/Sphinx/code.html

Reply:
Done